pkg/system/reconciler.go

```diff
@@ -287,6 +291,7 @@ func NewReconciler(
 	r.AdapterHPA.Name = r.Request.Name + "-hpav2"
 	r.BucketLoggingPVC.Name = r.Request.Name + "-bucket-logging-pvc"
 	r.BucketNotificationsPVC.Name = r.Request.Name + "-bucket-notifications-pvc"
+	r.SecretMetricsAuth.Name = "metrics-auth-secret"
```

Comment:
prefix with the CR name
```suggestion
	r.SecretMetricsAuth.Name = r.Request.Name + "-metrics-auth-secret"
```